BootloaderCommonPkg/Library/MmcAccessLib/SdMmcPciHci.c

```diff
@@ -1047,12 +1049,19 @@ BuildAdmaDescTable (
   Entries   = DivU64x32 ((DataLen + ADMA_MAX_DATA_PER_LINE - 1), ADMA_MAX_DATA_PER_LINE);
   TableSize = (UINTN)MultU64x32 (Entries, sizeof (SD_MMC_HC_ADMA_DESC_LINE));
   Trb->AdmaPages = (UINT32)EFI_SIZE_TO_PAGES (TableSize);
-  Trb->AdmaDesc  = AllocatePages (Trb->AdmaPages);
 
-  if (Trb->AdmaDesc == NULL) {
+  Status = IoMmuAllocateBuffer (
+    EFI_SIZE_TO_PAGES (TableSize),

```

Comment:
better update alignment for the second line to follow EDK2 coding style if there is other update.

Reply:
will do